openmc/mesh.py

```diff
@@ -665,6 +679,92 @@ def __repr__(self):
         string = super().__repr__()
         return string + '{: <16}=\t{}\n'.format('\tFilename', self.filename)
 
+    def write_data_to_vtk(self, filename, datasets, volume_normalization=True):
+        """Map data to the unstructured mesh element centroids
+           to create a VTK point-cloud dataset.
+
+        Parameters
+        ----------
+
+        filename : str
+            Name of the VTK file to write.
+        datasets : dict
+            Dictionary whose keys are the data labels
+            and values are the data sets.
+        volume_normalization : bool
+            Whether or not to normalize the data by the
+            volume of the mesh elements
+        """
+
+        if not _VTK:
+            raise RuntimeError("The VTK Python module is not installed.")
```

Comment:
Although it breaks from PEP8, I think a better way to handle this is to just put the import statement as the first thing in this method (we do this elsewhere). That way, we don't need to set the `_VTK` global. Also, I wouldn't include a try/except for an `ImportError` since having that propagate to the user tells them that they don't have the module installed.

Reply:
I like that approach better as well.